ios/Trifle/Sources/Digital Signature/Secure Enclave/SecureEnclaveDigitalSignatureKeyManager.swift

```diff
@@ -72,32 +73,53 @@ public class SecureEnclaveDigitalSignatureKeyManager
 
     // MARK: - Internal Methods (DigitalSignatureKeyManager)
 
-    internal func signingKey() throws -> SecureEnclaveSigningKey {
+    internal func createOrLoadSigningKey() throws -> SecureEnclaveSigningKey {
```

Comment:
I think we can keep the naming here to `signingKey` with the updated comments you had in the protocol